# HG changeset patch
# User ohair
# Date 1256840222 25200
# Node ID 8885b22565077236a927e824ef450742e434a230
# Parent  1602e8848bde3aa1c375390374998bfba46aa7b4
6896472: Missing libjsig.so causes jdk build failure
Reviewed-by: tbell, kvn, phh

diff -r 1602e8848bde -r 8885b2256507 make/java/redist/Makefile
--- openjdk.orig/jdk/make/java/redist/Makefile	Mon Oct 19 16:31:48 2009 -0700
+++ openjdk/jdk/make/java/redist/Makefile	Thu Oct 29 11:17:02 2009 -0700
@@ -211,7 +211,7 @@
 $(LIB_LOCATION)/$(KERNEL_LOCATION)/$(JVM_NAME): $(HOTSPOT_KERNEL_PATH)/$(JVM_NAME)
 	$(install-file)
 
-$(LIB_LOCATION)/$(LIBJSIG_NAME): $(HOTSPOT_SERVER_PATH)/$(LIBJSIG_NAME)
+$(LIB_LOCATION)/$(LIBJSIG_NAME): $(HOTSPOT_IMPORT_PATH)/$(ARCH_VM_SUBDIR)/$(LIBJSIG_NAME)
 	$(install-import-file)
 
 $(LIB_LOCATION)/$(CLIENT_LOCATION)/$(LIBJSIG_NAME) \
